build/pkgs/six/dependencies

```diff
@@ -1,4 +1,4 @@
- | $(PYTHON_TOOLCHAIN) $(PYTHON)
+ | pip $(PYTHON)
 
```

Comment:
`PYTHON_TOOLCHAIN` represents "a prerequisite to any pip-installed packages". It does not differentiate source and wheel python packages. Here changing it with just "pip" signals that `PYTHON_TOOLCHAIN` is only for source python packages.  Is it?

On the other hand, packages in `PYTHON_TOOLCHAIN` are installed anyway because of other packages. So it seems that the change has no real benefit. No?

Reply:
> changing it with just "pip" signals that `PYTHON_TOOLCHAIN` is only for source python packages. Is it?

Indeed. I have updated the documentation.